handsontable/src/plugins/mergeCells/mergeCells.js

```diff
@@ -135,6 +134,8 @@ export class MergeCells extends BasePlugin {
       }
     });
 
+    this.registerShortcut();
```

Comment:
I think that all places where you use `registerShortcut` should be renamed to plural version `registerShortcuts`. WDYT (the same with `unregisterShortcut`)?

Reply:
Only the `MergeCells` plugin has `registerShortcuts` method as it registers only one shortcut. But, for sure I can change it.

Reply:
Changed within 516f4f5ba92c1a6b5b6d908f485e186f533f506c.

Reply:
Changed within 516f4f5ba92c1a6b5b6d908f485e186f533f506c.